spec/features/search_crawler_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe "Search History Page" do
+  describe "crawler search" do
+    it "doesn't remember bot searches" do
+      page.driver.header('User-Agent', 'Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)')
+      visit root_path
+      fill_in "q", with: 'cat'
+      expect { click_button 'Search' }.not_to change { Search.count }
+    end
+  end
+end
```

Comment:
Why did you remove the part of this test that ensures searches are being saved for users? Would you please add it back?